### PR TITLE
13.0 mig pos order mgmt

### PR DESCRIPTION
**Pull request by ammarofficewla**
Migration in order management in v13

### CODE REVIEW

**Review by danimv5 — APPROVED** [review]
Functional review! LGTM

File: pos_order_mgmt/static/src/js/widgets.js [hunk 1]
```diff
@@ -180,13 +180,13 @@ odoo.define('pos_order_mgmt.widgets', function (require) {
...
-                    'XmlReceipt', {
-                        receipt: order.export_for_printing(),
+                    'OrderReceipt', {
                         widget: this,
                         pos: this.pos,
                         order: order,
+                        receipt: order.export_for_printing(),
                         orderlines: order.get_orderlines(),
-                        paymentlines: order.get_paymentlines(),
+                        paymentlinesf: order.get_paymentlines(),
```
Comment (legalsylvain) [inline pos_order_mgmt/static/src/js/widgets.js:189]: typo ? don't understand this change.

**Review by legalsylvain — COMMENTED** [review]
quick code review. LGTM. Just a question. 

thanks for the work.

**Review by eduardgm — APPROVED** [review]
Functional review, LGTM 👍🏻

**Comment by pedrobaeza** [comment]
Superseded by ##584

**Pull request closed without merging**